packages/devtools_app/test/common_widgets_test.dart

```diff
@@ -214,84 +214,79 @@ void main() {
 
   group('AreaPaneHeader', () {
     const titleText = 'The title';
-    const leftActionKey = Key('testLeftAction');
+    const leftActionText = 'The Left Action';
+    const centerActionText = 'The Center Action';
     const centerActionContainerKey = Key('scrollableCenterActionsContainer');
-    const centerAction = Text(
-      'Center Action',
-    );
-    const leftAction = Text(
-      'Left Action',
-      key: leftActionKey,
-    );
+    const centerAction = Text(centerActionText);
+    const leftAction = Text(leftActionText);
 
     setUp(() {
       setGlobal(IdeTheme, IdeTheme());
     });
 
-    group('with center actions', () {
-      testWidgets('onlyShowRightActions = true', (WidgetTester tester) async {
-        await tester.pumpWidget(
-          wrap(
-            const AreaPaneHeader(
-              title: Text(titleText),
-              leftActions: [leftAction],
-              scrollableCenterActions: [centerAction],
-              onlyShowRightActions: true,
-            ),
+    testWidgets(
+        'does NOT show center actions if no left or center actions present',
```

Comment:
is this accurate? We always want to show center actions if center actions are present, even if there are no left actions

Reply:
in this case both center and left actions  are absent, so this case was checking to make sure the center actions did not  show. I'll clean up the wording and the tests when i add the left right and center  :)